sql/src/test/java/io/crate/analyze/SetAnalyzerTest.java

```diff
@@ -71,6 +75,21 @@ public void testSet() throws Exception {
                 .put("stats.operations_log_size", 0)
                 .put("stats.jobs_log_size", 0)
                 .build()));
+
+        analysis = analyze("SET LOCAL stats.jobs_log_size TO 2");
+        assertThat(analysis.settings().toDelimitedString(','), is("stats.jobs_log_size=[2],"));
+
+        analysis = analyze("SET stats.jobs_log_size TO 2");
+        assertThat(analysis.settings().toDelimitedString(','), is("stats.jobs_log_size=[2],"));
```

Comment:
maybe split these tests into multiple test functions.
Makes it more clear what aspect is being tested and if something breaks it will also be easier to debug